lib/navigation/crosscheck.hpp

```diff
@@ -0,0 +1,22 @@
+#include <array>
+
+namespace hyped::navigation{
+
+  class Crosscheck{
+
+    public:
+
+      Crosscheck();
```

Comment:
None of the functions here take any arguments. How does the crosschecker (whatever it is) obtain data?

Reply:
TBD. From what I currently understand from @kshxtij, the data will be brought in through the use of callback functions which do not yet exist. Currently undecided as to how to deal with data once we have fresh readings somewhere in central navigation class. Might just pass directly to other functionalities or store as a class member and define some function to access it or maybe some other way.

Reply:
Not looking to have that sorted right now though. More just looking to get an outline done so that we can start splitting tasks between nav team and start implementing.

Reply:
Even when data is being passed in through callbacks, you need functions within your class that accept `std::function<something>` to actually pass in functions.

We can discuss this more tonight.